Amdad_form.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Your thoughts</title>
+        <link rel="stylesheet" href="form_style.css" >
+    </head>
+    <body>
+            <!-- Button group for navigation though website-->
+            <div class="btn-group">
+                <a href="Amdad_home.html">
+                    <button>Home</button>
+                </a>
+                <a href="About_amdad.html">
+                    <button>About Amdad</button>
+                </a>
+                <a href="Amdad_form.html">
+                    <button>Your thoughts?</button>
+                </a>
+            </div>
+            <!-- Form for user engagement-->
+            <h1>
+                Your thoughts?
+            </h1>
+            
+            <h2>
+                Here are some testimonials for Amdad. Do you agree?
+            </h2>
+
+            <form action="/signup" method="POST">
```

Comment:
Forms are used on almost every webpage so great that you have included one. Read up on forms and their attributes, we have a sign up set as action and "POST" as the method. Are they doing what you think they should be doing? If they are not doing anything, should they be in the code?